modules/openapi-generator/src/test/java/org/openapitools/codegen/typescript/typescriptnode/TypeScriptNodeClientOptionsTest.java

```diff
@@ -42,6 +42,7 @@ protected CodegenConfig getCodegenConfig() {
     protected void verifyOptions() {
         verify(clientCodegen).setSortParamsByRequiredFlag(Boolean.valueOf(TypeScriptNodeClientOptionsProvider.SORT_PARAMS_VALUE));
         verify(clientCodegen).setModelPropertyNaming(TypeScriptNodeClientOptionsProvider.MODEL_PROPERTY_NAMING_VALUE);
+        verify(clientCodegen).setParamNaming(TypeScriptNodeClientOptionsProvider.PARAM_NAMING_VALUE);
```

Comment:
nice!

but I also meant a regular unit test that checks the logic behind the new option, not just that it's properly supported by cli interface.

e.g. like this tests makes sure enumPropertyNaming is honored when formatting enums: https://github.com/OpenAPITools/openapi-generator/blob/2364dd7eb18d6154a89f5f638e88ba7fbbaa684e/modules/openapi-generator/src/test/java/org/openapitools/codegen/typescript/typescriptangular/TypeScriptAngularClientCodegenTest.java#L71

Reply:
thanks for the pointer, I updated the test accordingly.